nautobot/extras/tests/test_views.py

```diff
@@ -2419,31 +2419,21 @@ def test_delete_system_jobs_fail(self):
     def test_bulk_delete_system_jobs_fail(self):
         system_job_queryset = self.model.objects.filter(module_name__startswith="nautobot.")
         pk_list = system_job_queryset.values_list("pk", flat=True)[:3]
-        initial_count = self._get_queryset().count()
         data = {
             "pk": pk_list,
             "confirm": True,
             "_confirm": True,  # Form button
         }
         # Try bulk delete with delete job permission
-        self.add_permissions("extras.delete_job")
-        response = self.client.post(self._get_url("bulk_delete"), data, follow=True)
-        self.assertBodyContains(
-            response,
-            f"Unable to delete Job {system_job_queryset.first()}. System Job cannot be deleted",
-            status_code=403,
-        )
```

Comment:
Here too the changes you've made are invalidating the intent of this test case. May be another that needs to be migrated to `test_jobs.py` as a new test case?